src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -365,15 +356,8 @@ private HostControlInfo CreateHostControl(HostControlInfo jsonHostControlInfo)
             //If the Highlight area was defined in the json file then a rectangle will be highlighted in the Overlay
             if (jsonHostControlInfo.HighlightRectArea != null)
             {
-                popupInfo.HighlightRectArea = new HighlightArea()
-                {
-                    HighlightColor = jsonHostControlInfo.HighlightRectArea.HighlightColor,
-                    WidthBoxDelta = jsonHostControlInfo.HighlightRectArea.WidthBoxDelta,
-                    HeightBoxDelta = jsonHostControlInfo.HighlightRectArea.HeightBoxDelta,
-                    WindowName = jsonHostControlInfo.HighlightRectArea.WindowName,
-                    WindowElementNameString = jsonHostControlInfo.HighlightRectArea.WindowElementNameString,
-                    UIElementTypeString = jsonHostControlInfo.HighlightRectArea.UIElementTypeString
-                };
+                //We use the HighlightArea copy contrutor
```

Comment:
same here

Reply:
@QilongTang  type fixed in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12250/commits/c3995980180bb91f906ebbe6eb891924d0a584ea